chaos_genius/views/anomaly_data_view.py

```diff
@@ -413,7 +413,7 @@ def disable_anomaly(kpi_id):
             message = f"Disabled Analytics for KPI ID: {kpi_id}"
             status = "success"
         else:
-            message = f"Failed to Disable Anomaly because it is not enabled for KPI ID: {kpi_id}"
+            message = f"Failed to Disable Anomaly because it is not enabled for KPI ID: {kpi_id}"  # noqa: E501
```

Comment:
Nitpick: this string can be broken down into two lines instead of ignoring the error

Reply:
Done.